client/src/core/user-behavior.js

```diff
@@ -6,8 +6,6 @@ import {SubjectBehavior} from './user-input.js';
 import {Vector3, Euler} from 'three';
 
 // Speeds are in metres/s
-const groundSpeed = [5.0, 10.0];
-// const flySpeed = [5.0, 24.0];
 const lookSpeed = [2, 3.5];
 const turnSpeed = [2.0, 3.0];
```

Comment:
Import `speeds` from `user-config.js` and feed those two arrays with the matching values instead of using literal numbers again, this way those values will only be defined in one single place.